packages/flutter/lib/src/foundation/binding.dart

```diff
@@ -6,7 +6,7 @@ import 'dart:async';
 import 'dart:convert' show json;
 import 'dart:developer' as developer;
 import 'dart:io' show exit;
-import 'dart:ui' show saveCompilationTrace;
+import 'dart:ui' as ui show saveCompilationTrace, Window, window;
```

Comment:
please add a comment here to the effect of `// Before importing more of dart:ui, please read the README. Thanks.`

Reply:
there's a typo :-)